Compile error in pppm_disp_intel.cpp

```
../pppm_disp_intel.cpp: In constructor ‘LAMMPS_NS::PPPMDispIntel::PPPMDispIntel(LAMMPS_NS::LAMMPS*, int, char**)’:
../pppm_disp_intel.cpp:63:26: error: no matching function for call to ‘LAMMPS_NS::PPPMDisp::PPPMDisp(LAMMPS_NS::LAMMPS*&, int&, char**&)’
   PPPMDisp(lmp, narg, arg)
                          ^
In file included from ../pppm_disp_intel.h:27:0,
                 from ../pppm_disp_intel.cpp:21:
../pppm_disp.h:44:3: note: candidate: LAMMPS_NS::PPPMDisp::PPPMDisp(LAMMPS_NS::LAMMPS*)
   PPPMDisp(class LAMMPS *);
   ^~~~~~~~
../pppm_disp.h:44:3: note:   candidate expects 1 argument, 3 provided

../pppm_disp.h:42:7: note: candidate: constexpr LAMMPS_NS::PPPMDisp::PPPMDisp(const LAMMPS_NS::PPPMDisp&)  class PPPMDisp : public KSpace {
       ^~~~~~~~
../pppm_disp.h:42:7: note:   candidate expects 1 argument, 3 provided
```

--- a/src/USER-INTEL/pppm_disp_intel.cpp
+++ b/src/USER-INTEL/pppm_disp_intel.cpp
@@ -59,8 +59,7 @@ enum{FORWARD_IK, FORWARD_AD, FORWARD_IK_PERATOM, FORWARD_AD_PERATOM,
 
 /* ---------------------------------------------------------------------- */
 
-PPPMDispIntel::PPPMDispIntel(LAMMPS *lmp, int narg, char **arg) :
-  PPPMDisp(lmp, narg, arg)
+PPPMDispIntel::PPPMDispIntel(LAMMPS *lmp) : PPPMDisp(lmp)
 {
   suffix_flag |= Suffix::INTEL;
 
@@ -97,12 +96,9 @@ PPPMDispIntel::~PPPMDispIntel()
   memory->destroy(drho6_lookup);
 }
 
-
-
 /* ----------------------------------------------------------------------
    called once before run
 ------------------------------------------------------------------------- */
-
 
 void PPPMDispIntel::init()
 {

--- a/src/USER-INTEL/pppm_disp_intel.h
+++ b/src/USER-INTEL/pppm_disp_intel.h
@@ -31,7 +31,7 @@ namespace LAMMPS_NS {
 
   class PPPMDispIntel : public PPPMDisp {
   public:
-    PPPMDispIntel(class LAMMPS *, int, char **);
+    PPPMDispIntel(class LAMMPS *);
     virtual ~PPPMDispIntel();
     virtual void init();
     virtual void compute(int, int);

--- a/src/USER-INTEL/pppm_intel.cpp
+++ b/src/USER-INTEL/pppm_intel.cpp
@@ -57,7 +57,7 @@ enum{FORWARD_IK,FORWARD_AD,FORWARD_IK_PERATOM,FORWARD_AD_PERATOM};
 
 /* ---------------------------------------------------------------------- */
 
-PPPMIntel::PPPMIntel(LAMMPS *lmp, int narg, char **arg) : PPPM(lmp, narg, arg)
+PPPMIntel::PPPMIntel(LAMMPS *lmp) : PPPM(lmp)
 {
   suffix_flag |= Suffix::INTEL;
 

--- a/src/USER-INTEL/pppm_intel.h
+++ b/src/USER-INTEL/pppm_intel.h
@@ -34,7 +34,7 @@ namespace LAMMPS_NS {
 
 class PPPMIntel : public PPPM {
  public:
-  PPPMIntel(class LAMMPS *, int, char **);
+  PPPMIntel(class LAMMPS *);
   virtual ~PPPMIntel();
   virtual void init();
   virtual void compute(int, int);

